server/routerlicious/packages/protocol-definitions/src/sockets.ts

```diff
@@ -132,4 +132,9 @@ export interface IConnected {
      * that is likely to be more up-to-date.
      */
     checkpointSequenceNumber?: number;
+
+    /**
+     * Represents the version of document at server.
+     */
+    epoch?: string;
```

Comment:
This looks fine to me, but if this is not going to be generally implemented by other services, I'd rather not add it here and follow the suggestion I made in https://github.com/microsoft/FluidFramework/pull/4899/files re defining ODSP specific interface

Reply:
But we also added it in IConnect message. So adding here seems fine to me. It is optional so others  can avoid it?